doc/api/perf_hooks.md

```diff
@@ -432,103 +531,40 @@ console.log(h.percentile(50));
 console.log(h.percentile(99));
 ```
 
-### Class: Histogram
-<!-- YAML
-added: v11.10.0
--->
-Tracks the event loop delay at a given sampling rate.
-
-#### histogram.disable()
-<!-- YAML
-added: v11.10.0
--->
-
-* Returns: {boolean}
-
-Disables the event loop delay sample timer. Returns `true` if the timer was
-stopped, `false` if it was already stopped.
-
-#### histogram.enable()
-<!-- YAML
-added: v11.10.0
--->
-
-* Returns: {boolean}
-
-Enables the event loop delay sample timer. Returns `true` if the timer was
-started, `false` if it was already started.
-
-#### histogram.exceeds
-<!-- YAML
-added: v11.10.0
--->
-
-* {number}
-
-The number of times the event loop delay exceeded the maximum 1 hour event
-loop delay threshold.
-
-#### histogram.max
-<!-- YAML
-added: v11.10.0
--->
-
-* {number}
-
-The maximum recorded event loop delay.
-
-#### histogram.mean
-<!-- YAML
-added: v11.10.0
--->
-
-* {number}
-
-The mean of the recorded event loop delays.
-
-#### histogram.min
+## perf_hooks.monitorRequestWrapLatency()
 <!-- YAML
-added: v11.10.0
+added: REPLACEME
 -->
 
-* {number}
-
-The minimum recorded event loop delay.
-
-#### histogram.percentile(percentile)
-<!-- YAML
-added: v11.10.0
--->
-
-* `percentile` {number} A percentile value between 1 and 100.
-* Returns: {number}
-
-Returns the value at the given percentile.
-
-#### histogram.percentiles
-<!-- YAML
-added: v11.10.0
--->
-
-* {Map}
-
-Returns a `Map` object detailing the accumulated percentile distribution.
-
-#### histogram.reset()
-<!-- YAML
-added: v11.10.0
--->
-
-Resets the collected histogram data.
+* Returns: {Histogram}
 
-#### histogram.stddev
-<!-- YAML
-added: v11.10.0
--->
+Creates a `Histogram` object that reports the latency of libuv
+request handles. A request handle is an async resource within
+Node.js that tracks the state of an individual asynchronous
+operation (e.g. a file read, a DNS lookup, etc). The latency
+of a request is the length of time from when the request is
+passed off to the event loop to when the corresponding callback
+is invoked.
```

Comment:
I'm not sure if the wording here is really matching the implementation. When I first read this I was assuming the time from triggering a request till it starts processing in the uv worker thread is measured (pure libuv load dependent) but if I understand the implementation correct it actually measures the complete duration of the request (lib uv latency + actual IO time).
Therefore I think `duration` would be a better wording than `latency`.

I think a metric for the pure uv request latency would be also nice but not that easy to extract.

Reply:
Measuring that lower level from Node.js is going to be difficult. This PR takes the route of abstracting away those lower bits. The intent here is not to be uber-precise on the latency but to give a general health check of the event loop activity over time.